### PR TITLE
Create README

### PR DESCRIPTION
**Pull request by zachlatta**
@jonleung can you review this?

### CODE REVIEW

**Comment by YafahEdelman** [comment]
You might want to rephrase "Please don't open issues here unless you know what you're doing.". I assume an issue like a spelling mistake would be fine. I feel that people opening issues would only be a _problem_ if tons of people did which is unlikely, so a much less severe warning is fine.

**Comment by zachlatta** [comment]
Agreed, I just went ahead and removed it entirely.

**Comment by zachlatta** [comment]
Agreed—just removed the warning entirely.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+## Sprints
+
+hackEDU is adopting a sprint cycle. Each sprint gives us a manageable amount of
+work per person, to be completed within the cycle. It adds a heartbeat to our
+development. We will seldom add or drop work mid-sprint, though exceptions may
+emerge. This means incoming PRs or new issues may not be addressed until the
+next sprint. We will try to make sure things don't get backlogged for long
+periods of time.
+
+If you'd like to join a sprint, let `@zachlatta` know how much time + and what
```
Comment (maxwofford) [inline README.md]: This line was hard to follow.

EDIT: basically, what does "letting @zachlatta know how much time" mean?

**Comment by YafahEdelman** [comment]
"The sprint checkin is a quick sync-up" seems weird since sync-up is more confusing and jargon like than checkin.

**Comment by zachlatta** [comment]
@jonleung can you review and merge if you approve?
